### ISSUE TITLE
Global Community Search

### ISSUE THREAD

**Issue by ShadowJonathan** [mormaer/Mlem#224]
Possibly a tie-in with https://browse.feddit.de/

**Comment by gstevenson** [comment]
Isn’t this already possible? When you’re on the Feeds tab press the white header at the top and you get a community search. If I look for “cars” I get results on Lemmy.world, Lemmy.ml, Lemmy.ca…

**Comment by ShadowJonathan** [comment]
Yes, but these all use the knowledge of the local server, that feddit.de community list is an aggregate and crawled list, and we could possibly enhance our app more with this method

**Comment by BrooklynMan** [comment]
per the video convo, this will likely require further research wrt methods of subscribing from different instances (workarounds/hacks/etc), API version skew, etc. Much testing will be required before rollout. This feature will likely live in internal test build long before public rollout.

Excellent idea, but, due to LemmyAPI shortcomings may not be practical to implement right away. We may need to interact with Lemmy devs on this one.
